ci/pipelines/shared/jinja.variables.yml

```diff
@@ -99,7 +99,7 @@ tests:
   CALL_STACK_TIMEOUT: '4500'
```

Comment:
be sure to increase CALL_STACK_TIMEOUT by the same amount as you increased the job timeout

Reply:
Ooh, good to know!  Just increased it by 30 minutes.